app/main/views/frameworks.py

```diff
@@ -911,14 +911,19 @@ def view_contract_variation(framework_slug, variation_slug):
                 {'question': form['accept_changes'].label.text, 'input_name': 'accept_changes'}
             ]
 
+    supplier_name = supplier_framework['declaration']['nameOfOrganisation']
+    variation_content = content_loader.get_message(framework_slug, variation_content_name).filter(
+        {'supplier_name': supplier_name}
+    )
```

Comment:
While quite a lot better than the old way of finding the supplier name, this still seems a bit ad-hoc.  
How are we supposed to think about it? Do we just pass in whatever we need as needed?